### DISCUSSION TITLE
iOS and Android

### DISCUSSION THREAD

**Discussion by ghost** [vkbo/novelWriter#2008] [Ideas]
Any ideas about getting this onto iOS or Android at some point? Maybe there is already a discussion about this...?

**Comment by vkbo** [comment]
There are no plans to do that. I can barely maintain the MacOS release as it is, since I don't own any Apple devices.

As for Android, novelWriter is just plain Python code, so as long as Python and the Qt libraries are available, it should run. Not sure how well the UI operates on touch screens though.
